src/suparnaNJune20/Bank.java

```diff
@@ -0,0 +1,72 @@
+package suparnaNJune20;
+
+public class Bank {
+
+	static double atmBal=100000;
+	static int debitCnt, creditCnt, printBillCnt;
+	int debit,credit, printBill;
+	
+	void debitTransaction(int amt, String name) {
+		if (amt>atmBal)	{
+			System.out.println("No Sufficient BAl ");
+			return;
+		}
+		else {
+			atmBal-=amt;
+			debit++;
+			debitCnt++;
+			System.out.println(amt+ " Rs debited sucesfully by " + name);
+		}
+	}
+		void  creditTransaction (int amt, String name) {
+			 atmBal+=amt;
+				System.out.println(amt+ " Rs credited sucesfully by"+ name);
+				credit++;
+				creditCnt++;
+	}		
+		void  printBillTransaction() {
+			System.out.println(" Current Balance in Atm :"+atmBal);
+			printBill++;
+			printBillCnt++;
+		}
+	
+	void individualTransactionSumary(String custName) {
```

Comment:
Here this method not required custName parameter. If you want to print customer name then you should take as instance variable and should have setName() method for this.

Reply:
done